materialchips/src/main/ets/components/MainPage/InputChip.ets

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (C) 2022 Application Library Engineering Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@Component
+export struct InputChip {
+  @State delete: number= 1
+  @State tickSize: number= 0
+  @State radius: number= 0
+  @State checked: boolean = false
+  @State private model: InputChip.Model = undefined
+
+  build() {
+    Column() {
+      Row() {
+        Image(this.model.getImage())
```

Comment:
Add validation of null and undefined values on model. Recheck and add for all such case